core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -241,6 +287,44 @@ export class WorkflowEditorComponent implements AfterViewInit {
     return jointPaperOptions;
   }
 }
+/**/
+function getObservables(domItem: any) {
```

Comment:
For typescript method, define in the following format

`[public/private] [static?] [name] (): [return type] {`
`          `
`}` 

Also, prevent the use of type `any` because we would not know what type of method is passed as the parameter.